### ISSUE TITLE
[Lezhin] Not all pages from the chapter are downloading.

### ISSUE THREAD

**Issue by Megs108** [RedSquirrel87/manga-downloader#874]
Hi Red,

So it's the same issue as I've had previously off Lezhin where the progress bar in the programme continues to move however I'm left with an Error as not all the pages from the chapter managed to download. This problem was sort of solved by lessoning the pause (I say sort of as sometimes I've had to also go into the specific chapter, scroll through it, then exit back onto the chapter list and start the download again for it to download completely - for some reason this seems to help the programme download all the pages...). 

This time though I've had the pause between pages set to 0 seconds and am still not able to download the full chapter. Even after doing the extra bit I mentioned in the brackets above. I tried clearing the cookies and reloading your programme however the problem has persisted. The one time, even when the percentage on the progress bar was increasing, I couldn't find the folder in my documents and I know for sure I didn't save it anywhere else.

Here is the URL and jmv_log.txt file: 
https://www.lezhinus.com/en/library/comic/en-US/payback_en
[jvm_log.txt](https://github.com/user-attachments/files/18267543/jvm_log.txt)

It is chapter 66. (I also tried chapter 67 and had the same issue).
Thanks for helping me!
Meg.

**Comment by RedSquirrel87** [comment]
Thank you for reporting the issue, in the log I see a http 403 error that means the website has blocked your access. Does the problem happen with any chapter of that specific manga or only with chapters 66 and 67? Unfortunately, since these are paid chapters, I can't test them, do you know if it's still possible to get coins for free with videos or similar methods?

**Comment by Megs108** [comment]
I'll try another chapter from a different manga. 

The only other ways to get coins is to collect them over three days (you get 12 coins a day). With these though you'll have to "redeem" the first 2 days as late in the 24hr period of "redemption" as the coins only last 48hrs. This way the first day's coins are still available on the 3rd day. However this only applies to week days. The video section is currently unavailable so the only other way to try is from the free coin zones (found in the app) I only do the "label images for Ai one" (not sure if it'll be the same for you but I find this option is only available late afternoon for me, might differ for you...). These bonus coins last many months.

**Comment by Megs108** [comment]
Hi Red,
So I just tried the first chapter (which is free) from this manga; https://www.lezhinus.com/en/comic/scent
and it downloaded 100%.
I feel I must add that this chapter only has 53 pages however chapter 66 of the other manga I guessed to have around 120 pages maybe more. I only ever managed to get to page 82 on one of the tries. Perhaps the issue is to do with the number of pages the chapter has?

**Comment by RedSquirrel87** [comment]
Yes, indeed it could be a quantity issue because we have had problematic cases like this in the past, especially related to the expiration of the links. I think I need to rewrite the support for Lezhin so that the request for the access token of links isn't made all at once at the start of acquiring the links, but instead it should be done one at a time for each page. This way, these problems could be avoided once and for all. The issue, however, is that the program as it is now cannot handle such a change, and I'll have to modify quite a few things. So, I definitely won't be able to fix this in the new version I'll release tomorrow, but I'll work on it to improve it for future versions 👍

**Comment by Megs108** [comment]
Okay, that is good news! At least it's something that can be fixed :)

**Comment by RedSquirrel87** [comment]
v70.3 released, it includes the new download procedure that should fix this issue 👍
